mlem/core/requirements.py

```diff
@@ -337,6 +364,22 @@ def modules(self) -> List[str]:
     def expanded(self) -> "Requirements":
         return expand_requirements(self)
 
+    def _add_conda(self, requirement: CondaPackageRequirement):
```

Comment:
I guess that is the reason to put it here. We probably should move this logic into `Requirement` class so that requirements check their compatibility themselves

Reply:
But that's probably for another issue

Reply:
Let's do this later then